web/modules/custom/dpl_breadcrumb/src/Services/BreadcrumbHelper.php

```diff
@@ -99,6 +99,42 @@ public function getStructureFieldName(): string {
     return $this->structureFieldName;
   }
 
+  /**
+   * Get relevant categories field, as it may differ due to FieldInheritance.
+   *
+   * Events have a field_categories, but in reality, we want to use the
+   * event_categories field instead, as that's where the inheritance is set up.
+   */
+  public function getCategoriesFieldId(FieldableEntityInterface $entity): ?string {
+    if ($entity->hasField('event_categories')) {
+      return 'event_categories';
+    }
+
+    if ($entity->hasField('field_categories')) {
+      return 'field_categories';
+    }
+
+    return NULL;
+  }
```

Comment:
agreed, but, in reality, it would be nice if we had made sure that the inherited fields had the 'real' ID - e.g. `field_categories` like in nodes, instead of `event_categories`

But, that's a future, seperate 'problem' IMO.